File: Freedom/我的简历/首页/ResumeViewController.m
//  ResumeViewController.m
//  Freedom
//  Created by Super on 16/8/18.
//  Copyright © 2016年 Super. All rights reserved.
//
#import "ResumeViewController.h"
#import "ResumeListViewController.h"
#import "ResumeDetailViewController.h"
@interface ResumeViewCell:BaseTableViewCell
@end
@implementation ResumeViewCell
-(void)initUI{
}
-(void)setDataWithDict:(NSDictionary *)dict{
    
}
@end
@interface ResumeViewController ()<UITableViewDelegate,UITableViewDataSource>{
    UITableView *refTableView;
    UILabel *label;
    BaseScrollView *ResumeHomeScrollV;
}
@end
@implementation ResumeViewController
- (void)viewDidLoad {
    [super viewDidLoad];
    self.title = @"首页";
    NSArray *a = [NSArray arrayWithObjects:@{@"pic":PuserLogo,@"name":@"我的简历",@"url":ResumeURL},@{@"pic":PuserLogo,@"name":@"微信小程序1",@"url":WeChatApplet1},@{@"pic":PuserLogo,@"name":@"微信小程序2",@"url":WeChatApplet2},
                  @{@"pic":PuserLogo,@"name":@"微页1",@"url":MicroPage1},@{@"pic":PuserLogo,@"name":@"微页2",@"url":MicroPage2},@{@"pic":PuserLogo,@"name":@"微页3",@"url":MicroPage3}, nil];
    NSArray *b = [NSArray arrayWithObjects:@{@"pic":PuserLogo,@"name":@"我的简历",@"url":ResumeURL},@{@"pic":PuserLogo,@"name":@"微信小程序1",@"url":WeChatApplet1},@{@"pic":PuserLogo,@"name":@"微信小程序2",@"url":WeChatApplet2},
                  @{@"pic":PuserLogo,@"name":@"微页1",@"url":MicroPage1},@{@"pic":PuserLogo,@"name":@"微页2",@"url":MicroPage2},@{@"pic":PuserLogo,@"name":@"微页3",@"url":MicroPage3}, nil];
    NSArray *c = [NSArray arrayWithObjects:@{@"pic":PuserLogo,@"name":@"我的简历",@"url":ResumeURL},@{@"pic":PuserLogo,@"name":@"微信小程序1",@"url":WeChatApplet1},@{@"pic":PuserLogo,@"name":@"微信小程序2",@"url":WeChatApplet2},
                  @{@"pic":PuserLogo,@"name":@"微页1",@"url":MicroPage1},@{@"pic":PuserLogo,@"name":@"微页2",@"url":MicroPage2},@{@"pic":PuserLogo,@"name":@"微页3",@"url":MicroPage3}, nil];
    NSArray *lists = @[a,b,c];
    
    NSArray *titles = @[@"我的成长史",@"我的作品",@"我的经历"];
    NSArray *controllers = @[@"ResumeListViewController",@"ResumeListViewController",@"ResumeListViewController"];
    ResumeHomeScrollV = [BaseScrollView sharedContentTitleViewWithFrame:CGRectMake(0, 0, APPW, APPH-TabBarH) titles:titles controllers:controllers inViewController:self];
    @weakobj(ResumeHomeScrollV);
    ResumeHomeScrollV.selectBlock = ^(NSInteger index, NSDictionary *dict) {
        ResumeListViewController *con = weak_ResumeHomeScrollV.contentScrollView.pushDelegateVC.childViewControllers[index];
        con.listArray = lists[index];
    };
}
@end

File: Freedom/我的简历/首页/ResumeViewController.swift
//
//  ResumeViewController.swift
//  Freedom
//
//  Created by Super on 2018/5/15.
//  Copyright © 2018年 Super. All rights reserved.
//
import UIKit
import XExtension
import XCarryOn
class ResumeViewCell:BaseCollectionViewCell{
    override func initUI() {
    }
}
class ResumeViewController: ResumeBaseViewController {
    var ResumeHomeScrollV:BaseScrollView!
    override func viewDidLoad() {
        super.viewDidLoad()
        title = "首页"
        let a = ["我的简历","微信","微信小程序2","微页1","微页2","微页3"]
        let b = ["我的简历","微页1","微页3","微信小程序1","微信小程序2","微页2"]
        let c = ["微信小程序1","我的简历","微信小程序2","微页1","微页2","微页3"]
        let titles = ["我的成长史","我的作品","我的经历"];
        let con1 = ResumeListViewController()
        let con2 = ResumeListViewController()
        let con3 = ResumeListViewController()
        con1.listArray = a
        con2.listArray = b
        con3.listArray = c
        let controllers = [con1,con2,con3];
        ResumeHomeScrollV = BaseScrollView(contentTitleView: CGRect(x: 0, y: 0, width: APPW, height: APPH-TopHeight), titles: titles, controllers: controllers, in: self)
        ResumeHomeScrollV.selectBlock = {(index,dict) in
            
        }
//        view.addSubview(ResumeHomeScrollV)
    }
}
